.release/ci.hcl

```diff
@@ -14,7 +14,7 @@ project "consul-ecs" {
     repository = "consul-ecs"
     release_branches = [
       "main",
-      "release/0.5.x",
+      "release/0.6.x",
```

Comment:
Because this is going into main, it should include all release branches. We can actually change this to be `release/**` now.

Reply:
:+1: Great catch Paul! I've updated as suggested.